cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -257,7 +257,94 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             };
         }
     });
+    TimedExaminationPreferenceEditor = AbstractEditor.extend({
+        templateName: 'timed-examination-preference-editor',
+        className: 'edit-settings-timed-examination',
 
+        events : {
+            'change #id_timed_examination': 'timedExamination',
```

Comment:
Nit: our best practice is to always use classes, not ids, although I see that there are other examples with ids here.